Sources/UrlFormEncoding/UrlFormDecoder.swift

```diff
@@ -10,21 +10,37 @@ public final class UrlFormDecoder: Decoder {
   public private(set) var codingPath: [CodingKey] = []
   public var dataDecodingStrategy: DataDecodingStrategy = .deferredToData
   public var dateDecodingStrategy: DateDecodingStrategy = .deferredToDate
-  public var parsingStrategy: ParsingStrategy = .accumulatePairs
+  public var parsingStrategy: ParsingStrategy = .accumulateValues
```

Comment:
Don't really know why I named it the other way. Was dealing with a lot of pairs at the time.